siemens_data.py

```diff
@@ -0,0 +1,74 @@
+import csv
+import sys
+import pandas as pd
+from datetime import datetime
+
+""" Class for reading and manipulating Siemens data from transformed CSVs. 
+	Contains API for accessing data, as well as function for reading the CSV's. 
+"""
+
+class SiemensData:
+
+	data = None
+	datetimes = []
+
+	# Open and store transformed CSV data as class pd object. 
+	def read_csv(self, filename): 
+		self.data = pd.read_csv(filename, dtype=object)
+
+	def find_data_for_room(self, room_name):
```

Comment:
This is great!! I like that you first specify room and then have access to all the equipment of the room.  This is a good format and makes the data in the csv much more accessible